source/addonHandler/__init__.py

```diff
@@ -688,7 +722,12 @@ def __init__(self, input, translatedInput=None):
 		@param translatedInput: translated manifest input
 		@type translatedInput: file-like object
 		"""
-		super(AddonManifest, self).__init__(input, configspec=self.configspec, encoding='utf-8', default_encoding='utf-8')
+		super(

```

Comment:
Since we are at Python 3 now you can safely omit parameters when calling `super` like this:
`super().__init__(param1, param2)`